guide/additional-info/changes-in-v12.md

```diff
@@ -159,7 +186,7 @@ The section headers for breaking changes will be named after the v11 classes/met
 
 <p class="danger">While this list has been carefully crafted, it may be incomplete! If you notice pieces of missing or inaccurate data, we encourage you to [submit a pull request](https://github.com/Danktuary/Making-Bots-with-Discord.js)!</p>
 
-## Breaking changes
+## Breaking Changes and Deletions
```

Comment:
Adding `and Deletions` here might be unnecessary because it's still breaking changes